styles/octopus.css

```diff
@@ -0,0 +1,57 @@
+.octopus {
+    padding-bottom: 84px;
+
+    background-image:
+    url("../images/bubble.png"),
+    url("../images/Group_bubls.png"),
+    linear-gradient(252.43deg, #D45B78 0.01%, #4B3862 101.01%);
+    background-repeat: 
+    no-repeat,
+    no-repeat,
+    no-repeat;
+    background-position: 
+    left 128px bottom 109px,
+    right 27px bottom 31px,
+    top 0 left 0;
+}
+
+.octopus__container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+
+    height: calc(100vh -84px);
```

Comment:
![image](https://user-images.githubusercontent.com/34751890/176017983-8246fba4-9b0a-4d12-8e1e-93fd515ae56c.png)
this style does not work